linera-execution/src/evm/revm.rs

```diff
@@ -586,6 +688,32 @@ where
         Self { module, db }
     }
 
+    /// Executes the transaction. If needed instantiate the contract
+    fn init_transact_commit_tx_data(
+        &mut self,
+        ch: Choice,
+        vec: &[u8],
+    ) -> Result<ExecutionResultSuccess, ExecutionError> {
+        // In case of a not yet instanstated application, we need first to create it.
```

Comment:
Isn't the application necessarily already instantiated, in Linera's sense? It's about initializing the EVM contract, isn't it?

Reply:
Yes, that is that. Linera instantiated, but not according to the EVM.
So, I changed the comment.